app/Http/Controllers/QueryController.php

```diff
@@ -42,7 +53,7 @@ public function nowOpenAction(GetQueryRequest $request, Service $service, Channe
             $start = new Carbon($testDateTime);
         }
 
-        return $this->getResponse($request, $start, $end, $service, $channel, true);
+        return $this->getResponse($request, $start, $end, $service, $channel,self::CALENDAR_LENGTH_UNDEFINED,true);
```

Comment:
missing space after $channel, and self::CALENDAR_LENGTH_UNDEFINED,

See https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#43-methods